tests/test_io.py

```diff
@@ -1690,3 +1691,175 @@ def test_csv_options_cs2(self):
         simple_alias = Dummy_FMUModelCS2([("x", "y")], os.path.join(file_path, "files", "FMUs", "XML", "CS2.0", "NegatedAlias.fmu"), _connect_dll=False)
         self._run_negated_alias(self, simple_alias)
     """
+
+class TestResultDymolaBinary:
```

Comment:
It seems like we only test that we can retrieve trajectories. Can we add checks so that the expected trajectories are returned?